### PR TITLE
Add CreateOutifts page and Header

### PR DESCRIPTION
**Pull request by happysituations**

### CODE REVIEW

File: .babelrc [hunk 1]
```diff
@@ -1,3 +1,4 @@
 {
-  "presets": ["env", "react"]
+  "presets": ["env", "react"],
+  "plugins": ["transform-object-rest-spread"]
```
Comment (skoshkarli) [inline .babelrc]: what does this do?

File: src/client/components/CreateOutfits/CreateOutfits.jsx [hunk 2]
```diff
@@ -0,0 +1,111 @@
+import React from 'react';
+import { withRouter } from 'react-router-dom';
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
+import { Grid, Card } from 'semantic-ui-react';
+// import { ??? } from '../actions/createOutfitsActions';
```
Comment (skoshkarli) [inline src/client/components/CreateOutfits/CreateOutfits.jsx]: please remove

File: src/client/components/CreateOutfits/CreateOutfits.jsx [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+import Axios from 'axios';
+import DropdownSearch from '../DropdownSearch.jsx';
+import MyClosetFilters from '../MyCloset/MyClosetFilters.jsx';
+
+var items = [
```
Comment (skoshkarli) [inline src/client/components/CreateOutfits/CreateOutfits.jsx]: should we use the example file we have with data?

File: src/client/components/CreateOutfits/CreateOutfits.jsx [hunk 4]
```diff
@@ -0,0 +1,111 @@
...
+            </Grid.Column>
+          </Grid.Row>
+        </Grid>
+      </div>
+    );
+  }
+}
+
+const mapStateToProps = state => ({
+  // weather: state.closetBoard.weather
```
Comment (skoshkarli) [inline src/client/components/CreateOutfits/CreateOutfits.jsx]: lets remove map state to props and map dispatch to props and the connect below if we are not using it along with the imports

File: src/client/index.jsx [hunk 5]
```diff
@@ -25,20 +27,32 @@ import {
   withRouter,
 } from 'react-router-dom';
 
+const DefaultLayout = ({ component: Component, ...rest }) => {
+  return (
+    <Route {...rest} render={matchProps => (
```
Comment (skoshkarli) [inline src/client/index.jsx]: what does this do?

File: src/client/components/CreateOutfits/CreateOutfits.jsx [hunk 6]
```diff
@@ -0,0 +1,111 @@
...
+    this.onDrop = this.onDrop.bind(this);
+  }
+
+  onDragStart(event, id) {
+    event.dataTransfer.setData("id", id);
+  }
+
+  onDrop(event) {
+    let id = event.dataTransfer.getData("id");
+    console.log(id);
```
Comment (hcnimi) [inline src/client/components/CreateOutfits/CreateOutfits.jsx]: console.log!

File: src/client/components/Header.jsx [hunk 7]
```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import { Menu } from 'semantic-ui-react';
+import { Link } from 'react-router-dom';
+
+class Header extends React.Component {
```
Comment (hcnimi) [inline src/client/components/Header.jsx]: can make presentational

File: .babelrc (hunk 1 above)
Comment (happysituations) [reply .babelrc]: the the ...rest in DefaultLayout

File: src/client/components/CreateOutfits/CreateOutfits.jsx (hunk 2 above)
Comment (happysituations) [reply src/client/components/CreateOutfits/CreateOutfits.jsx]: Got rid of it, for now

File: src/client/components/CreateOutfits/CreateOutfits.jsx (hunk 3 above)
Comment (happysituations) [reply src/client/components/CreateOutfits/CreateOutfits.jsx]: yes.... :D, you're right, I'll try to do this before presentations

File: src/client/components/Header.jsx (hunk 7 above)
Comment (happysituations) [reply src/client/components/Header.jsx]: done

**Comment by skoshkarli** [comment]
🚢 it

**Pull request merged**